docs/details/linux.md

```diff
@@ -0,0 +1,39 @@
+title: Zero Install for Linux
+
+The Linux version of Zero Install extends the cross-platform core [Zero Install .NET](../developers/dotnet-api.md) with a GUI and various OS-specific integrations.
```

Comment:
This is not correct - it does not extend the .NET version. It is an alternative that does not require .NET at all.

Reply:
Oh, sorry. That's the result of too hasty copy&paste. :( Will fix shortly.

Reply:
Fixed.